### PR TITLE
feat(timestate): Add schema support for timeline slices.

### PR DESCRIPTION
**Pull request by bradh**
In support of https://github.com/ngageoint/opensphere/issues/467

### CODE REVIEW

File: src/main/xsd/state/1.0.0/types.xsd [hunk 1]
```diff
@@ -138,6 +138,26 @@
     </xs:restriction>
   </xs:simpleType>
 
+	<xs:complexType name="TimeIntervalType">
```
Comment (schmidtk) [inline src/main/xsd/state/1.0.0/types.xsd]: I don't wish to elicit a holy war here, but... replace tabs with spaces please.

File: src/main/xsd/state/1.0.0/state.xsd [hunk 2]
```diff
@@ -1681,6 +1681,15 @@
         </xs:annotation>
       </xs:element>
 
+      <xs:element name="slices" type="fade:SlicesType"
+        minOccurs="0" maxOccurs="1">
+        <xs:annotation>
+          <xs:documentation xml:lang="en" xml:space="preserve">
+            One or more time slice intervals.
```
Comment (schmidtk) [inline src/main/xsd/state/1.0.0/state.xsd]: The documentation should explain the behavior of slice intervals. Maybe something along the lines of:
```
Defines an interval less than 24 hours in which data is loaded for each day in the load interval.
```
Feel free to spruce that up if it seems unclear.

File: src/main/xsd/state/1.0.0/types.xsd (hunk 1 above)
Comment (bradh) [reply src/main/xsd/state/1.0.0/types.xsd]: Urgh. Sorry, copied it in from xmlspy and didn't notice the difference in vscode...

File: src/main/xsd/state/1.0.0/state.xsd (hunk 2 above)
Comment (bradh) [reply src/main/xsd/state/1.0.0/state.xsd]: Added revised docs here, and added missing xml annotations for new types.

**Pull request merged**